rundeckapp/grails-app/views/scm/performAction.gsp

```diff
@@ -191,8 +192,9 @@
 
                                         <div class="checkbox col-sm-12">
                                                 <g:checkBox name="deletePaths" value="${path}"
+                                                            id="deletePaths${path}"
```

Comment:
the value of `path` is likely to be like `project/X/some-uuid.xml` which is not going to be a valid HTML ID attribute value.  In the `<g:each` you can add a `status="somevar"` which will create a variable `somevar` as the index of the item in the list... then you could use `id="deletedPath${somevar}"` for the id